src/taproot.rs

```diff
@@ -148,22 +126,13 @@ pub mod test_helpers {
 
     #[allow(non_snake_case)]
     fn sign_params<RNG: RngCore + CryptoRng, Signer: traits::Signer>(
-        msg: &[u8],
+        _msg: &[u8],
```

Comment:
Can this be removed since its not used? Or is this to minimize downstream changes?

Reply:
Yeah `stacks-sbtc` is fine with this removed.